Dockerfile

```diff
@@ -8,9 +8,9 @@
 FROM alpine:3.14
 RUN apk add --no-cache --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community "skopeo>=1.2.0"
 
-COPY k8s-image-swapper /
```

Comment:
@charlie-haley Just curious, is there any technical reason behind this change?

Reply:
There wasn't actually, I think it was left over from local debugging, I'll revert the file